src/main/Duke.java

```diff
@@ -0,0 +1,165 @@
+                System.out.println(String.format(
+                    "%s      %s", Commands.ADD.toString(), deadline.identify()));
+                tasks.add(deadline);
+                System.out.println(String.format("    Now you have %d tasks in the list.", tasks.size()));
+                break;
+            case "todo":
+                //Exception catching
+                if (processedCommand.length <= 1) {
+                    new DukeException();
+                    break;
+                }
+
+                //Create Todo
+                ToDo todo = new ToDo(processedCommand[1]);
+                System.out.println(String.format(
+                    "%s      %s", Commands.ADD.toString(), todo.identify()));
+                tasks.add(todo);
+                System.out.println(String.format("    Now you have %d tasks in the list.", tasks.size()));
+                break;
+            case "event":
+                //Exception catching
+                if (processedCommand.length <= 1) {
+                    new DukeException();
+                    break;
+                } else if (!processedCommand[1].contains("/by")) {
+                    new DukeException();
+                    break;
+                }
+
+                //Input processing
+                String[] inputForEventConstructor = processedCommand[1].split("/by", 2);
+                inputForEventConstructor[0] = inputForEventConstructor[0].trim();
+                LocalDate eventDate = null;
+                try {
+                    eventDate = LocalDate.parse(inputForEventConstructor[1].trim());
+                } catch (DateTimeParseException e) {
+                    new DukeException();
+                    break;
+                }
+
+                //Create event
+                Event event = new Event(inputForEventConstructor[0], eventDate);
+                System.out.println(String.format(
+                    "%s      %s", Commands.ADD.toString(), event.identify()));
+                tasks.add(event);
+                System.out.println(String.format("    Now you have %d tasks in the list.", tasks.size()));
+                break;
+            case "delete":
+                //Exception catching
+                if (processedCommand.length <= 1) {
+                    new DukeException();
+                    break;
+                }
+                try {
+                    taskIndex = Integer.parseInt(processedCommand[1]) - 1;
+                } catch (NumberFormatException e) {
+                    new DukeException();
+                    break;
+                }
+
+                Task deletedTask = tasks.remove((int)taskIndex);
+                System.out.println(String.format(
+                    "%s      %s\n    Now you have %d tasks in the list",
+                    Commands.DELETE.toString(), deletedTask.identify(), tasks.size()));
+                break;
+            default:
+                new DukeException();
```

Comment:
can try abstracting this more (make it more OOP) through the use of a commands and class!